corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -597,11 +600,22 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             return {
                 title: this.options.title.trim(),
                 description: DOMPurify.sanitize(description),
-                sidebarEnabled: this.options.sidebarEnabled,
+                sidebarEnabled: this.sidebarEnabled,
                 grouped: Boolean(this.collection.find(c => c.has("groupKey"))),
             };
         },
 
+        onRender() {
+            var submitButton = this.ui.submitButton;
+            if (this.sidebarEnabled) {
+                if (sessionStorage.submitDisabled === false || sessionStorage.submitDisabled === "false") {
```

Comment:
You could do `JSON.parse(sessionStorage.submitDisabled) === false` rather than two conditionals.